pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
     "Topic :: Multimedia :: Sound/Audio",
 ]
 dependencies = [
-    "pipecat-ai>=0.0.84",
+    "pipecat-ai>=0.0.85",
```

Comment:
Note: update this in case the corresponding Pipecat changes don't make it out in this version.